spec/controllers/account/comments_controller_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+RSpec.describe Account::CommentsController, type: :controller do
+  render_views
+
+  let!(:project) { create(:project) }
+  let!(:task) { create(:task, project: project) }
+  let!(:user) { create(:user) }
+  let!(:comment_for_task) { create(:comment, :for_task, body: 'Body', user: user, commentable: task) }
+  let!(:comment_for_project) { create(:comment, :for_project, body: 'Body', user: user, commentable: project) }
```

Comment:
something is not right.. why do u need `:for_project` along w/ `commentable: project) ?